README.md

```diff
@@ -1,40 +1,35 @@
 # Anonymous Plugin [![CircleCI branch](https://img.shields.io/circleci/project/github/bakurits/mattermost-plugin-anonymous/key-pair-generation.svg)](https://circleci.com/gh/bakurits/mattermost-plugin-anonymous) [![codecov](https://codecov.io/gh/bakurits/mattermost-plugin-anonymous/branch/key-pair-generation/graph/badge.svg)](https://codecov.io/gh/bakurits/mattermost-plugin-anonymous)
```

Comment:
These links are incorrect for the master branch

Reply:
updated. sorry for that